app/crud.py

```
@@ -0,0 +1,65 @@
+        ) from e
+
+
+async def get_todo(db: AsyncSession, todo_id: int) -> ToDo:
+    try:
+        result = await db.execute(select(ToDo).where(ToDo.id == todo_id))
+        return result.scalar_one()
+    except NoResultFound:
+        raise HTTPException(status_code=404, detail=f"ToDo with ID:{todo_id} not found")
+    except Exception as e:
+        raise HTTPException(
+            status_code=500, detail=f"Error retrieving ToDo with ID:{todo_id}: {e}"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)